finalPrep/modZeroHero.js

```diff
@@ -2,40 +2,73 @@
 
 // Declare two variables - heroName AND specialAbility - set to strings
 
+var heroName = 'Super Sonic';
+var specialAbility = 'Lightening Speed';
+
 // Declare two variables - greeting AND catchphrase
 //   greeting should be assigned to a string that uses concatenation to include the heroName
 //   catchphrase should be assigned to a string that uses interpolation to include the specialAbility
 
+var greeting = ('Hello ' + heroName);
+var catchphrase = (`The ${specialAbility} hero you need!`);
+
 // Declare two variables - power AND energy - set to integers
+var power = 40;
+var energy = 150;
 
 // Declare two variables - fullPower AND fullEnergy
 //   fullPower should multiply your current power by 500
 //   fullEnergy should add 150 to your current energy
+var fullPower = (power*500);
+var fullEnergy = (energy+150);
 
 // Declare two variables - isHuman and identityConcealed - assigned to booleans
+var isHuman = false;
+var identityConcealed = true;
 
 
 // Declare two variables - archEnemies AND sidekicks
 //   archEnemies should be an array of at least 3 different enemy strings
 //   sidekicks should be an array of at least 3 different sidekick strings
+var archEnemies = ['Laser Guy', 'Racoon Man', 'Geese'];
+var sidekicks = ['Mighty Hedgehog', 'Hot Hamster', 'Flying Squirral'];
 
 // Print the first sidekick to your console
-
+console.log(sidekicks[0]);
 // Print the last archEnemy to the console
+console.log(archEnemies[2]);
 
 // Write some code to add a new archEnemy to the archEnemies array
+archEnemies.push('Banana Peels');
 
 // Print the archEnemies array to console to ensure you added a new archEnemy
-
+console.log(archEnemies);
 // Remove the first sidekick from the sidekicks array
+sidekicks.shift();
 
 // Print the sidekicks array to console to ensure you added a new sidekick
-
+console.log(sidekicks);
 // Create a function called assessSituation that takes three arguments - dangerLevel, saveTheDay, badExcuse
 //   - dangerLevel should be an integer
-//   - saveTheDay should be a string a hero would say once they save the day 
+//   - saveTheDay should be a string a hero would say once they save the day
 //   - badExcuse should be a string a hero would say if they are too afraid of the dangerLevel
 
+function assessSituation(dangerLevel, saveTheDay, badExcuse)  {
+  console.log(dangerLevel, saveTheDay, badExcuse);
+
+  if(dangerLevel > 50) {
+    console.log(badExcuse);
+  }
+  else if(dangerLevel < 10) {
+    console.log('Meh. Hard pass.');
+  }
+  else{
+    console.log(saveTheDay);
+  }
+
```

Comment:
What if the dangerLevel was equal to 50? What if the `dangerlLevel == 10`? Account for these cases too.